### DISCUSSION TITLE
Where do I find the implementation of /sellclaimblocks?

### DISCUSSION THREAD

**Discussion by Dutch-0** [GriefPrevention/GriefPrevention#2311] [Q&A] [answered]
Like the title says. Somehow I was not able the find any code pointing to that command using the search.

**Comment by Jikoo** [comment, marked as answer]
https://github.com/GriefPrevention/GriefPrevention/blob/f5f850a98d43b6ed194e65636ce368dc7f762392/src/main/java/me/ryanhamshire/GriefPrevention/GriefPrevention.java#L1905-L1978

**Reply by Jikoo** [reply]
You probably were looking in the master branch, which no longer features economy support out of the box since 2d4b89ecbea63d5f650ac904949aa04526d27722.

**Reply by Dutch-0** [reply]
Ah thank you. So buying and selling claim blocks will be unsupported in future versions? Is there a date scheduled for the new release?

**Reply by Jikoo** [reply]
17.0.0, no date that I'm aware of. 16.X is on life support - no features, only fixes, and even then only if someone cares enough to write the fix.

**Reply by Dutch-0** [reply]
Based on the code in main, creating a simple plugin which would replicate /buyclaim and /sellclaim should be doable for me.
For now I will write a fix for my current issue and create a PR. 
Thank you very much :+1:
